packages/console/src/pages/AuthStatusChecker/utils.ts

```diff
@@ -0,0 +1,213 @@
+import { DOMAIN_WHITELIST, STORAGE_ACCESS_TEST_KEY } from './consts';
+import type { CheckAdminTokenMessage } from './types';
+import { AuthMessageType } from './types';
+
+// Common two-level TLDs (country code + generic)
+// We can add more TLDs here if needed.
+const COMMON_TWO_LEVEL_TLDS = new Set([
```

Comment:
Can we use [psl](https://www.npmjs.com/package/psl)?

Reply:
Good call, updated.